README.md

```diff
@@ -11,7 +11,7 @@
 
 <!-- ## Live Demo (if available) -->
 
-<!-- [Live Demo Link](https://livedemo.com) -->
+[Live Demo Link](https://emmiiorji.github.io/my_new_portfolio/)
```

Comment:
**Your project working perfectly! Congratulations! 🥳🎉**

- [ ] [OPTIONAL] It is always a good idea to keep the project as clean as possible. I would strongly recommend to you delete and ignore unused or extra files from your Readme.MD file. As you can see in this image...

<img width="518" alt="Screen Shot 2022-04-05 at 5 07 14 AM" src="https://user-images.githubusercontent.com/69011963/161653592-3b4c5e38-aac0-4952-a8fd-7b4b5c9978b5.png">

`.vscode` is extra in here!